templates/layout.php

```diff
@@ -17,8 +17,8 @@
           <a class="main-header__logo" href="index.php">
               <img src="../img/logo.svg" width="160" height="39" alt="Логотип компании YetiCave">
           </a>
-          <form class="main-header__search" method="get" action="https://echo.htmlacademy.ru" autocomplete="off">
-              <input type="search" name="search" placeholder="Поиск лота">
+          <form class="main-header__search" method="get" action="search.php" autocomplete="off">
+              <input type="search" name="search" placeholder="Поиск лота" value="<?= htmlspecialchars($search); ?>">
```

Comment:
```suggestion
              <input type="search" name="search" placeholder="Поиск лота" value="<?= htmlspecialchars($search ?? ''); ?>">
```
Это поле имеет смысл заполнять только на странице поиска, так что давай не будем с каждого скрипта передавать сюда пустую строку